<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sign_up</class>
 <widget class="QMainWindow" name="sign_up">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1311</width>
    <height>741</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1311</width>
      <height>741</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/photos/playground.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="lineWidth">
     <number>5</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_password">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>200</y>
      <width>241</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tempus Sans ITC</family>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Password : </string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_name">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>320</y>
      <width>191</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tempus Sans ITC</family>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Name :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_phone">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>440</y>
      <width>221</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tempus Sans ITC</family>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Phone :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_email">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>560</y>
      <width>211</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tempus Sans ITC</family>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Email :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_skull_pirate">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/photos/Pirate skull.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_password">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>220</y>
      <width>501</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tempus Sans ITC</family>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_name">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>340</y>
      <width>501</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tempus Sans ITC</family>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_phone">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>460</y>
      <width>501</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tempus Sans ITC</family>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_email">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>580</y>
      <width>501</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tempus Sans ITC</family>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_skull_pirate_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>320</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/photos/Pirate skull.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_skull_pirate_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>440</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/photos/Pirate skull.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_skull_pirate_5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>560</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/photos/Pirate skull.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_back_skull">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>60</y>
      <width>291</width>
      <height>361</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/photos/skull-king-pirate.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_save">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>440</y>
      <width>211</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tempus Sans ITC</family>
      <pointsize>50</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/photos/playground.png);</string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_userrname">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>261</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tempus Sans ITC</family>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Username :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_username">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>100</y>
      <width>501</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tempus Sans ITC</family>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_skull_pirate_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/photos/Pirate skull.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cancel">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>560</y>
      <width>211</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tempus Sans ITC</family>
      <pointsize>40</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/photos/playground.png);</string>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_username_massage">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>170</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tempus Sans ITC</family>
      <pointsize>17</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_password_massage">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>290</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tempus Sans ITC</family>
      <pointsize>17</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_name_massage">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>420</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tempus Sans ITC</family>
      <pointsize>17</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_phone_massage">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>540</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tempus Sans ITC</family>
      <pointsize>17</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_email_massage">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>650</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tempus Sans ITC</family>
      <pointsize>17</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>lbl_background</zorder>
   <zorder>lbl_userrname</zorder>
   <zorder>lbl_password</zorder>
   <zorder>lbl_name</zorder>
   <zorder>lbl_phone</zorder>
   <zorder>lbl_skull_pirate</zorder>
   <zorder>line_username</zorder>
   <zorder>line_password</zorder>
   <zorder>line_name</zorder>
   <zorder>lbl_skull_pirate_2</zorder>
   <zorder>lbl_email</zorder>
   <zorder>line_phone</zorder>
   <zorder>line_email</zorder>
   <zorder>lbl_skull_pirate_3</zorder>
   <zorder>lbl_skull_pirate_4</zorder>
   <zorder>lbl_skull_pirate_5</zorder>
   <zorder>lbl_back_skull</zorder>
   <zorder>btn_save</zorder>
   <zorder>btn_cancel</zorder>
   <zorder>lbl_username_massage</zorder>
   <zorder>lbl_password_massage</zorder>
   <zorder>lbl_name_massage</zorder>
   <zorder>lbl_phone_massage</zorder>
   <zorder>lbl_email_massage</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
